besu/src/main/java/org/hyperledger/besu/cli/BesuCommand.java

```diff
@@ -21,6 +21,7 @@
 import static org.hyperledger.besu.cli.DefaultCommandValues.getDefaultBesuDataPath;
```

Comment:
@macfarla this file needs to be reviewed

Reply:
LGTM